README.md

```diff
@@ -208,7 +208,9 @@ methods:
   and the enum member value at index 1.
 - `Enum.has(E, key)` - Returns `true` if the the \[\[EnumMembers]] internal slot of `E` contains `key`.
 - `Enum.hasValue(E, value)` - Returns `true` if the \[\[EnumMembers]] internal slot of `E` contains a
-  member whose value on `E` corresponds to`value`.
+  member whose value on `E` corresponds to `value`.
+- `Enum.getName(E, value)` - Gets the first name in the \[\[EnumMembers]] internal slot of `E` whose
+  value on `E` corresponds to `value`.
 - `Enum.format(E, value)` - Calls the @@formatEnum method of `E` with argument `value`.
 - `Enum.parse(E, value)` - Calls the @@parseEnum method of `E` with argument `value`.
 - `Enum.create(members)` - Creates an _enum object_ using the property keys and values of `members`
```

Comment:
`Enum.create` should probably take an optional second argument to specify a base enum type.

Reply:
That's unnecessary. The base enum type is only used to determine the auto value for enum members. `Enum.create` takes an object literal whose property values are used directly.

Reply:
I don't think I'd do an overload, but if there's a strong desire for this mechanism, we could consider adding an `Enum.from(iterable, baseType)`:

```js
const dynamicEnum = Enum.from(memberNames, Number);
```

Reply:
Yeah, that looks nice. Then I can make enums of keys of classes and stuff. ;)